### ISSUE TITLE
clj-commons.humanize/datetime fails with java.time.LocalDateTime in cljs

### ISSUE THREAD

**Issue by roterski** [clj-commons/humanize#39]
In cljs env with the following versions:
````clojure
org.clj-commons/humanize {:mvn/version "1.0"}
thheller/shadow-cljs     {:mvn/version "2.25.5"}
tick/tick                {:mvn/version "0.6.2"}
````
running any one of:
```clojure
(h/datetime)
(h/datetime (js/Date.))
(require '[tick.core :as t])
(h/datetime (t/now))
```
results in the same exception:
````clojure
(h/datetime)
:repl/exception!
; 
; Execution error (ExceptionInfo) at (<cljs repl>:1).
; unable to coerce to java.time.LocalDateTime
````

**Comment by danielcompton** [comment]
This is because of the change to use clic.java.time instead of cljs-time. I think `clj-commons.humanize.time-convert/coerce-to-local-date-time` needs to be updated to support JavaScript Dates.

(https://github.com/clj-commons/humanize/pull/40)

**Comment by hlship** [comment]
Fixed by #41
